run_service.py

```diff
@@ -0,0 +1,228 @@
+from pathlib import Path
+from operate.cli import OperateApp
+from operate.types import (
+    LedgerType,
+    ServiceTemplate,
+    ConfigurationTemplate,
+    FundRequirementsTemplate,
+    OnChainUserParams,
+)
+from operate.account.user import UserAccount
+import time
+import sys
+import getpass
+import requests
+from halo import Halo
+from termcolor import colored
+
+OLAS_BALANCE_REQUIRED_TO_BOND = 10_000_000_000_000_000_000
+OLAS_BALANCE_REQUIRED_TO_STAKE = 10_000_000_000_000_000_000
+XDAI_BALANCE_REQUIRED_TO_BOND = 10_000_000_000_000_000
+SUGGESTED_TOP_UP_DEFAULT = 50_000_000_000_000_000
+SUGGESTED_SAFE_TOP_UP_DEFAULT = 500_000_000_000_000_000
+MASTER_WALLET_MIMIMUM_BALANCE = 1_000_000_000_000_000_000
+WARNING_ICON = colored('\u26A0', 'yellow')
+OPERATE_HOME = Path.cwd() / ".operate2"
+
+TEMPLATE = ServiceTemplate(
+    {
+        "name": "trader_service",
+        "hash": "bafybeicxdpkuk5z5zfbkso7v5pywf4v7chxvluyht7dtgalg6dnhl7ejoe",
+        "image": "",
+        "description": "trader service",
+        "configuration": ConfigurationTemplate(
+            {
+                "rpc": "",
```

Comment:
Is there a way to force for example, subgraph api key and rpc url to be necessary?